src/core/modelchecking.cpp

```diff
@@ -46,6 +49,12 @@ std::shared_ptr<storm::modelchecker::CheckResult> modelCheckingHybridEngine(std:
     return storm::api::verifyWithHybridEngine<DdType, ValueType>(env, model, task);
 }
 
+std::shared_ptr<storm::modelchecker::CheckResult> checkIntervalMdp(std::shared_ptr<storm::models::sparse::Mdp<storm::Interval>> mdp, CheckTask<double> const& task, storm::Environment& env) {
+    env.solver().minMax().setMethod(storm::solver::MinMaxMethod::ValueIteration);
```

Comment:
I would prefer not to magically change the environment and have the user be responsible for this. Also because I hope to soon have extensions to more standard settings.

Reply:
We currently have no bindings to change the environment from stormpy. I will add some bindings. Users will then have to make sure to use the right solver for intervals (at least for now).